volatility3/framework/plugins/windows/mftscan.py

```diff
@@ -427,11 +455,11 @@ def get_requirements(cls):
     @classmethod
     def parse_first_data_records(
         cls,
-        record_map: Dict[int, Tuple[str, int, int]],
-        mft_record: interfaces.objects.ObjectInterface,
-        attr: interfaces.objects.ObjectInterface,
+        record_map: DefaultDict[str, MFTRecord],
```

Comment:
```suggestion
        record_map: Dict[str, MFTRecord],
```
Again, the type doesn't need to be DefaultDict, I think just Dict should be fine here?